Makefile

```diff
@@ -41,7 +41,7 @@ clean:
 	(rm -rf build)
 
 build-app:
-	make -C SampleApps/$(APP_NAME) build-shallow
+	#make -C SampleApps/$(APP_NAME) build-shallow
```

Comment:
we should fix this in dev

Reply:
Yes I made an issue yesterday. https://github.com/adobe/aepsdk-edgeidentity-ios/issues/37